workspaces/quickstart/plugins/quickstart/src/components/QuickstartContent/QuickstartItem.tsx

```diff
@@ -150,7 +159,21 @@ export const QuickstartItem = ({
           }}
         >
           <ListItem>
-            <ListItemText primary={item.description} />
+            <ListItemText
+              primary={
+                item.descriptionKey
+                  ? (() => {
+                      const translation = t(
+                        item.descriptionKey as keyof typeof t,
+                        {},
+                      );
+                      return translation !== item.descriptionKey
+                        ? translation
+                        : item.description;
+                    })()
+                  : item.description
+              }
+            />
```

Comment:
Personally I would move this also up, before the TSX code.

Calculating a `title` and a `description` and then just use it here.